packages/app-project/src/components/Announcements/components/GenericAnnouncement/components/NarrowGenericAnnouncement/NarrowGenericAnnouncement.js

```diff
@@ -1,9 +1,18 @@
 import { Box } from 'grommet'
 import { CloseButton, Markdownz } from '@zooniverse/react-components'
 import { bool, string, func } from 'prop-types'
+import styled from 'styled-components'
 
 import markdownzComponents from '../../helpers/markdownzComponents'
 
+// Spaces the paragraphs created by Markdown a little more nicely.
+// Box.gap='large' doesn't seem to work.
+const SpacedBox = styled(Box)`
+  p {
+    margin: 0.25em 0;
+  }
+`
```

Comment:
I think this is possible with Box props in Grommet, but I can't remember how off the top of my head.